Assets/MixedRealityToolkit/SDK/ExampleScenes.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
```

Comment:
Wouldn't we want the examples to be in MixedRealityToolkit-Examples?

Reply:
Let's chat about it when you get online.  It could be argued that the SDK has it's own examples and the MRTK might have it's own examples (for non-SDK use)
I see the SDK being it's own "thing" entirely imho.